src/api.ts

```diff
@@ -296,7 +296,7 @@ async function getRemoteBranches(): Promise< Map< string, string > > {
 	timing( 'git.refresh', Date.now() - start );
 
 	try {
-		const branchesReferences = ( await repo.getReferences( git.Reference.TYPE.OID ) ).filter(
+		const branchesReferences = (await repo.getReferences( )).filter(
```

Comment:
The enum is gone (https://www.nodegit.org/api/repository/#getReferences).
Probably related to "There are now native implementations of the following: Repository.prototype.getReferences: Retrieves all references on async worker." from nodegit 0.25.0 (https://github.com/nodegit/nodegit/blob/master/CHANGELOG.md#fixed)

https://github.com/nodegit/nodegit/issues/1737 also mentions the enum being deprecated.